docs/python101/python101-spec.tex

```diff
@@ -33,7 +33,8 @@
 \newcommand{\defmin}{\stackrel{{\scriptscriptstyle \mathsf{min}}}{=}}
 \newcommand{\colcol}{\coloncolon}
 
-\newcommand{\yield}{\colcol\!>}
+\newcommand{\yield}{\uparrow}
```

Comment:
No, I don't like the up and down arrows ...  For type checking I don't want any direction I really want `::`  because that's the standard notation (or just `:`) and  for type synthesis the arrow has to go from the term to the type, so a right arrow ... in a way I still prefer `::>`.   I find that a left arrow is not good because type checking is not term synthesis. I don't like the existing notations...

Reply:
Okay, then. I kind of see what you mean. Let's go with what was originally planned.